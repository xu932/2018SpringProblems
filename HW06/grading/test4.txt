name:Natividad 
name:Cosme 
name:Premier 
name:Camp 
name:Rennick 
name:Copelynn 
name:Anir 
name:Jaxston 
name:Luann 
name:Nataley 
name:Addelynn 
name:Javan 
name:Finnick 
name:Sayuri 
name:Meridian 
name:Aizen 
name:Malhar 
name:Juandavid 
name:Jocee 
name:Jayln 
name:Evleen 
name:Samanvi 
name:Amirjon 
name:Samatar 
name:Evalisse 
name:Sal 
name:Sandy 
name:Jadah 
name:Maxwel 
name:Virgil 
name:Srinidhi 
name:Benoit 
name:Airess 
name:Pippi 
name:Kol 
name:Joelie 
name:Rayhan 
name:Viera 
name:Gayatri 
name:Braelyn 
name:Marianna 
name:Lanah 
name:Aristotelis 
name:Colten 
name:Shani 
name:Yeshia 
name:Yafa 
name:Abdulai 
name:Dedrick 
name:Tessla 
name:Azrael 
name:Chloe 
name:Breighlyn 
name:Sadieann 
name:Numa 
