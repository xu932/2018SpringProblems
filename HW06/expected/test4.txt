id:23 major:CS enrollment:No 
id:25 major:CS enrollment:No 
id:58 major:CS enrollment:No 
id:151 major:CS enrollment:No 
id:176 major:CS enrollment:No 
